src/midi-player.ts

```diff
@@ -69,7 +159,10 @@ export class MidiPlayer implements IMidiPlayer {
 
         events
             .filter(({ event }) => MidiPlayer._isSendableEvent(event))
-            .forEach(({ event, time }) => this._midiOutput.send(this._encodeMidiMessage(event), start + time));
+            .forEach(({ event, time }) => {
+                this._midiOutput.send(this._encodeMidiMessage(event), start + time);
+                this._latest = Math.max(this._latest!, start + time);
```

Comment:
What about only doing this for node on events? I might be wrong but I guess we can ignore everything else.